bin/cloudnet/tosca/network_diagrams.py

```diff
@@ -27,9 +27,13 @@
     # Target directory where network diagrams are generated.
     Generator.TARGET_DIRECTORY: "Results/NetworkDiagrams",
     # Network capability types.
-    "linkable_capability_types": ["tosca.capabilities.network.Linkable"],
+    "linkable_capability_types": ["tosca.capabilities.nfv.VirtualLinkable"],
```

Comment:
Previous value shall be kept else network diagram generation for examples/OASIS_TOSCA_1_2 will break.
tosca.capabilities.nfv.VirtualLinkable is specific to ETSI NFV SOL001 so this value could be configured into the tosca2cloudnet.yaml file of SOL001-related example.
The main idea is to move configuration outside the code.

Reply:
This is understandable. @JLCoulin shall we still get this defaults for the web front end ? In this case, maybe we can provide a default config file as a commandline parameter ?

Reply:
So we have to provide a tosca2cloudnet.yaml file for the example [TOSCA_1.3_NFV_Topology](https://github.com/Orange-OpenSource/Cloudnet-TOSCA-toolbox/tree/master/examples/TOSCA_1.3_NFV_Topology).
This is not the case currently.